app/views/declaration/summary/item_section.scala.html

```diff
@@ -265,7 +265,7 @@
                     content = Text(messages("declaration.summary.items.item.supplementaryUnits"))
                 ),
                 value = Value(
-                    content = Text(commodityMeasure.supplementaryUnits.getOrElse(""))
+                    content = Text(commodityMeasure.supplementaryUnits.getOrElse("No"))
```

Comment:
Not sure, but this might require it's own `messages` key-value pair (considering Welsh).

Reply:
Yeah, good spot. :+1: